workspaces/services/guacamole/.env.sample

```diff
@@ -0,0 +1,11 @@
+TRE_ID="__CHANGE_ME__"
```

Comment:
It shouldn't need all these, as they are loaded from the `devops/.env` and `templates/core/.env` folders.

Only WORKSPACE_ID, and GUACAMOLE_IMAGE_TAG ?